### DISCUSSION TITLE
Set initial form values

### DISCUSSION THREAD

**Discussion by Sicria** [esamattis/react-zorm#56] [General]
Is it possible to provide an object of initial form values that can be used to set the initial value of the inputs?

For example editing an item you'd want to prefill some form values.

**Comment by esamattis** [comment]
There's no special APIs for that. You must just use the `defaultValue` props in the input components.

**Comment by esamattis** [comment]
Actually you could use something like [updateForm](https://github.com/findkit/update-form) in a useEffect hook to set initial values from a object.
